pydux/create_store.py

```diff
@@ -81,17 +73,17 @@ def dispatch(action):
             raise ValueError('Actions must have a "type" property. '
                              'Have you misspelled a constant?')
 
-        if is_dispatching[0]:
+        if self.is_dispatching[0]:
             raise Exception('Reducers may not dispatch actions.')
 
         try:
-            is_dispatching[0] = True
-            current_state[0] = reducer(current_state[0], action)
+            self.is_dispatching[0] = True
+            self.current_state[0] = self.current_reducer[0](self.current_state[0], action)
```

Comment:
Not sure if this was originally a bug. To me it looks like it wouldn't dispatch properly when the reducer was replaced the way it is on master.